main.go

```diff
@@ -234,6 +234,23 @@ var (
 			printer.PrintStatus(state)
 		},
 	}
+
+	listAppsCmd = &cobra.Command{
+		Use:   "apps",
```

Comment:
Just checking because I dont know.

Is it apps or is it docs? And are we consistent?

Reply:
It's apps nowadays.. but it used to be docs :)